.github/workflows/maven.yml

```diff
@@ -51,4 +64,50 @@ jobs:
           '-Dorg.unicode.cldr.unittest.web.KeepDb=true'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+  check:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: false
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Download cldr.jar
+        uses: actions/download-artifact@v2
+        with:
+          name: cldr-tools
+          path: tools/java/target/
+      - name: run CLDR console check
+        run: java -DCLDR_DIR=$(pwd) -jar tools/java/target/cldr.jar check -S common,seed -e -z FINAL_TESTING 
+  deploy:
+    if: github.repository == 'unicode-org/cldr' && github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs:
+      - build
+      - check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download cldr-apps.war
+      uses: actions/download-artifact@v2
+      with:
+        name: cldr-apps
+    - name: Deploy to cldr-smoke (experimental)
```

Comment:
How is this triggered and how does it relate to other ways of deployment?

Reply:
This is the deploy to smoketest on merge that currently is done with ant.  It's triggered the same way.